assets/js/notifications.js

```diff
@@ -0,0 +1,173 @@
+const publicVapidKey =
+  "BAK9aUUilxzljIZVaKm8gjt6MWYsXQFbhluMMCFCddHbWug4_H48Q4XtbCwBSPZ9V3wcNUGr92twrEbmGMyABKU";
+const firebaseConfig = {
+  apiKey: "AIzaSyCZ8gbWs9FLlzw93BOj_OQzlWwd_F1N-IY",

```

Comment:
Are these `apikey` and `publicVapidKey`  are secret keys for an app? Then we should take these from environment variables and not put them here

Reply:
```publicVapidKey``` and ```apikey``` will be taken from firebase project.. as stated this is yet to be done, currently I have created firebase project using my account

Reply:
Ah I got that. But are these some secret tokens that shouldn't be directly exposed in code ? Then we should be setting it in some env file or conf file and use it here.